client/sagas/execute-challenge-saga.js

```diff
@@ -99,29 +99,34 @@ export default function executeChallengeSaga(action$, getState) {
       const { files } = state.challengesApp;
```

Comment:
This might be better placed as a transformer. Is there a reason you did it directly in the build?